### ISSUE TITLE
Interval support

### ISSUE THREAD

**Issue by hexpunk** [hexpunk/tbd#2]
A multi-day span of time should be able to be represented in the month view from https://github.com/hexpunk/tbd/issues/1. Overlapping intervals are out of scope for this ticket.  Assume no intervals will overlap for now.

# Acceptance Criteria

- [ ] Based on an interval of time, given as a start date and an end date, a continuous block will appear over the dates in the calendar.
  - Ideally, this is a separate block element that actually exists and spans the Day components within the Month component.  If this isn't easily done, I can consider marking Day components if they sit within a span.  The rest of the ACs are assuming that's not necessary.
- [ ] There should be a visual indication reinforcing the concept of when an interval spans between multiple calendar weeks.
  - This happens on both sides of the break, both on Saturday and Sunday.
  - There should be a distinctive difference compared to if an interval ends on a Saturday and a new interval starts on Sunday.
  - Options could be arrows of some kind pointing towards the outside of the interval box. Or maybe a jagged or curvy border like axis breaks in charts.
- [ ] The tab stop of this component should be just _before_ the tab stop for the first Day component within this interval, since the component sits visually on top of the Day component.

**Comment by hexpunk** [comment]
This issue is blocked by https://github.com/hexpunk/tbd/issues/1
